styleguide/_themes/derek/scss/components/ticker.scss

```diff
@@ -17,11 +17,11 @@ $ticker-span-bg: rgba(200, 0, 0, .5);
 }
 
 .ticker-container {
-  @include make-container;
+  @include make-container-fluid;
```

Comment:
i explained this elsewhere, but basically we're forcing width restrictions on row, instead of container.